types/redux-first-router/index.d.ts

```diff
@@ -152,9 +183,15 @@ export interface Options {
     onAfterChange?<S>(dispatch: Dispatch<S>, getState: StateGetter): void;
     onBackNext?<S>(dispatch: Dispatch<S>, getState: StateGetter): void;
     restoreScroll?(history: History): ScrollBehavior;
+    initialDispatch?: boolean;
+    querySerializer?: QuerySerializer;
     navigators?: NavigatorsConfig;
 }
 
+// Params and Payload differ from Flow definition, as they can be arbitrarily shaped
+export type Params = any;
+export type Payload = any;
+
```

Comment:
You're actually right, I'm sorry. However, the problem with `any` remains. A possible solution would be doing something along the lines of

```ts
export type Payload = {
    [key: string]: any
}